nion/swift/DocumentController.py

```diff
@@ -749,6 +749,8 @@ def import_file(self) -> None:
         filter += ";;All Files (*.*)"
         import_dir = self.ui.get_persistent_string("import_directory", "")
         paths, selected_filter, selected_directory = self.get_file_paths_dialog(_("Import File(s)"), import_dir, filter)
+        if len(paths) == 0:
```

Comment:
Similar ... `if len(paths) > 0:` and then put lines 754+755 inside the `if` statement.